api/nodes/views.py

```diff
@@ -125,16 +125,78 @@ def fetch_from_waterbutler(self):
+
+        name           type               description
+        ---------------------------------------------------------------------------------
+        title          string             title of project or component
+        description    string             description of the node
+        category       string             node category, must be one of the allowed values
+        date_created   iso8601 timestamp  timestamp that the node was created
+        date_modified  iso8601 timestamp  timestamp when the node was last updated
+        tags           array of strings   list of tags that describe the node
+        registration   boolean            has this project been registered?
+        collection     boolean            is this node a collection of other nodes?
+        dashboard      boolean            is this node visible on the user dashboard?
+        public         boolean            has this node been made publicly-visible?
+
+    ##Links
+
+    See the [JSON-API spec regarding pagination](http://jsonapi.org/format/1.0/#fetching-pagination).
+
+    ##Actions
+
+    ###Creating New Nodes
+
+        Method:        POST
+        URL:           links.self
+        Query Params:  <none>
+        Body (JSON):   {
+                         "data": {
+                           "type": "nodes", # required
+                           "attributes": {
+                             "title":       {title},         # mandatory
+                             "category":    {category},      # mandatory
+                             "description": {description},   # optional
+                             "tags":        [{tag1}, {tag2}] # optional
+                           }
+                         }
+                       }
+        Success:       201 CREATED + node representation
+
+    New nodes are created by issuing a POST request to this endpoint.  The `title` and `category` fields are
+    mandatory. `category` must be one of the [permitted node categories](/v2/#osf-node-categories).  All other fields
+    not listed above will be ignored.  If the node creation is successful the API will return a 201 response with the
+    respresentation of the new node in the body.  For the new node's canonical URL, see the `links.self` field of the
+    response.
+
+    ##Query Params
+
+    + `page=<Int>` -- page number of results to view, default 1
+
+    + `filter[<fieldname>]=<Str>` -- fields and values to filter the search results on.
+
+    Nodes may be filtered by their `title`, `category`, `description`, `public`, `registration`, or `tags`.  `title`,
+    `description`, and `category` are string fields and will be filtered using simple substring matching.  `public` and
+    `registration` are booleans, and can be filtered using truthy values, such as `true`, `false`, `0`, or `1`.  Note
+    that quoting `true` or `false` in the query will cause the match to fail regardless.  `tags` is an array of simple strings.
+
+    #This Request/Response
```

Comment:
It's not a comment, it's a markdown header.  It's there as a visual marker to distinguish the docs from the response section that BAPI generates.  I felt that the docs and the response sections ran together too much.